ompi/include/mpi.h.in

```diff
@@ -920,6 +921,48 @@ enum {
   MPI_T_PVAR_CLASS_GENERIC
 };
 
+/*
+ * MPIT callback safety levels
+ *
+ * Values are set in configure.ac for consistency with mca_base_event.h
+ */
+#undef OPAL_MCA_BASE_CB_REQUIRE_NONE
```

Comment:
I'm puzzled by how is this supposed to work ? They are not surrounded by @ so they will not be replaced by configure, and they are undefined just before. What values would the give to the enum ?

Reply:
I missed th